projects/Mallard/src/helpers/webview.ts

```diff
@@ -216,15 +217,6 @@ const makeJavaScript = (topPadding: number) => html`
         document.addEventListener('scroll', debounce(onScroll), {
             passive: true,
         })
-
-        const openLightbox = index => {
```

Comment:
nice simplification reading this! Makes it much easier to follow what's happening